lib/shelley/test/data/balanceTx/pingPong_2/golden

```diff
@@ -53,12 +53,12 @@
  2.700000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2700000)], minimumSelectionAmount = Coin 4050000})))
- 2.750000,0.612050,0.612050
- 2.800000,0.612050,0.612050
- 2.850000,0.612050,0.612050
- 2.900000,0.612050,0.612050
- 2.950000,0.612050,0.612050
- 3.000000,0.612050,0.612050
+ 2.750000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2750000)], minimumSelectionAmount = Coin 4125000})))
+ 2.800000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2800000)], minimumSelectionAmount = Coin 4200000})))
+ 2.850000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2850000)], minimumSelectionAmount = Coin 4275000})))
+ 2.900000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2900000)], minimumSelectionAmount = Coin 4350000})))
+ 2.950000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 2950000)], minimumSelectionAmount = Coin 4425000})))
+ 3.000000,ErrBalanceTxSelectAssets (ErrSelectAssetsSelectionError (SelectionCollateralErrorOf (SelectionCollateralError {largestCombinationAvailable = fromList [((TxIn {inputId = Hash "00000000000000000000000000000000", inputIx = 0},Address "`\177\229\224\251t\200l\128\USdhA\224|\219B\223\139\130\239<\228\229|\181A.w"),Coin 3000000)], minimumSelectionAmount = Coin 4500000})))
```

Comment:
This appears odd. When we have 3 ada we get an error saying that we need at least 4.5 ada. But with 3.05 ada, balancing succeeds.

The reason is:
1. The minimum fee is around 2 ada (≈0.5 from the size, and ≈1.5 for the _maximum_ ex unit costs)
2. With an excess of _just_ 1 ada, this is right at the limit of where coin-selection can't create a change output. Instead it returns a higher "selection delta" / "fee".
3. The collateral requirement is based on the actual `fee`, not the `minfee`. With `collateralPercentage: 150%`, this results in a 3 ada *1.5 = 4.5 ada collateral requirement.